README.md

```diff
@@ -33,5 +33,36 @@ async def main():
 
 
 asyncio.run(main())
+```
+
+#### WoCurtain (Curtain 3)
+
+```python
+import asyncio
+from pprint import pprint
+
+from switchbot import GetSwitchbotDevices
+from switchbot.devices import curtain
+
+
+async def main():
+    # get the BLE advertisement data of all switchbot devices in the vicinity
+    advertisement_data = await GetSwitchbotDevices().discover()
+
+    for i in advertisement_data.values():
+        pprint(i)
+        print() # print newline so that devices' data is separated visually
+
+    # get the BLE device (via its address) and construct a curtain device
+    ble_device = advertisement_data["9915077C-C6FD-5FF6-27D3-45087898790B"].device
```

Comment:
Please make `"9915077C-C6FD-5FF6-27D3-45087898790B"` a named constant so they know what it is